apps/meteor/server/services/video-conference/service.ts

```diff
@@ -1190,7 +1193,7 @@ export class VideoConfService extends ServiceClassInternal implements IVideoConf
 
 	private async addUserToDiscussion(rid: IRoom['_id'], uid: IUser['_id']): Promise<void> {
 		try {
-			await Room.addUserToRoom(rid, { _id: uid }, undefined, { skipAlertSound: true });
+			await Room.addUserToRoom(rid, { _id: uid }, undefined, { skipAlertSound: true, skipSystemMessage: true, createAsHidden: true });
```

Comment:
In the code you're assinging `false` to alert when `createAsHidden` is true, so passing `skipAlertSound` is not needed.